uiowa/uiowa
# Event Locations Enhancements

Resolves https://github.com/uiowa/uiowa/issues/6402

Same as https://github.com/uiowa/uiowa/pull/6511 but without autofill 
stuff.

- New fields and replicate in the default event feature.
- Events backend overview for OnIowa only.

<img width="1229" alt="Screenshot 2023-06-19 at 9 34 50 AM" src="https://github.com/uiowa/uiowa/assets/4663676/683d5e63-1adf-42b3-9d0c-ada0a2db3d85">


# Test

```
ddev composer install && ddev blt ds --site=oniowa.uiowa.edu && ddev drush @oniowa.local config:import --source ../config/sites/oniowa.uiowa.edu --partial -y && ddev drush @oniowa.local config-split:activate site -y && ddev drush @oniowa.local uli node/add/event
```
- Create an event node with an address location including the ~company~ location name field 👀, geolocation and driving directions. geolocation should not display on full node.
- Go to admin/content. You should see a new "Events" link next to Overview and Moderated content. That view should have basic event information displayed that the customer can use to identify similar locations for use with replicate.
- Replicate? Yeah. Try it on the event you just created.
- Check admissions/iswf for clean imports. Clean imports should mean they have the new fields and are ready to go. Those sites should also have replicate available due to the update hook.
```
ddev blt ds --site=admissions.uiowa.edu && ddev blt ds --site=iowasummerwritingfestival.uiowa.edu
```
